src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -0,0 +1,150 @@
+    @observable centeredChildRelativeTop: number;
+    list: HTMLElement;
+    scrollTop: number;
+
+    componentDidMount() {
+        window.addEventListener('blur', this.requestClose);
+        window.addEventListener('resize', this.requestClose);
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('blur', this.requestClose);
+        window.removeEventListener('resize', this.requestClose);
+    }
+
+    componentDidUpdate() {
+        window.requestAnimationFrame(() => {
+            if (this.list) {
+                this.list.scrollTop = this.scrollTop;
+            }
+        });
+    }
+
+    requestClose = () => {
+        if (this.props.isOpen && this.props.onRequestClose) {
+            this.props.onRequestClose();
+        }
+    };
+
+    @computed get dimensions(): ?OverlayListDimensions {
+        if (!this.offsetHeight || !this.offsetHeight || !this.centeredChildRelativeTop) {
+            return null;
+        }
+        const positioner = new OverlayListPositioner(
+            this.offsetHeight,
+            this.offsetWidth,
+            this.centeredChildRelativeTop,
+            this.props.anchorTop,
+            this.props.anchorLeft,
+            this.props.anchorWidth,
+            this.props.anchorHeight,
+        );
+        return positioner.getCroppedDimensions();
+    }
+
+    readCenteredChildRelativeTop = (option: Option) => {
+        if (option) {
+            window.requestAnimationFrame(action(() => {
+                this.centeredChildRelativeTop = option.getOffsetTop();
+            }));
+        }
+    };
+
+    readOffsetDimensions = (list: HTMLElement) => {
+        this.list = list;
+        if (!list || this.offsetWidth || this.offsetHeight) {
+            return;
+        }
+        window.requestAnimationFrame(action(() => {
+            this.offsetWidth = list.offsetWidth;
+            this.offsetHeight = list.offsetHeight;
+        }));
+    };
+
+    render() {
+        let style = {visibility: 'hidden'};
+        const dimensions = this.dimensions;
+        if (dimensions) {
+            style = OverlayListPositioner.dimensionsToStyle(dimensions);
+            this.scrollTop = dimensions.scrollTop;
+        }
+
+        return (
+            <div>
+                <Portal isOpened={this.props.isOpen}>
+                    <div className={overlayListStyles.container}>
+                        <ul
+                            ref={this.readOffsetDimensions}
+                            style={style}
+                            className={overlayListStyles.list}>
+                            {this.renderChildrenWithFocusSet()}
+                        </ul>
+                    </div>
+                </Portal>
+                <Backdrop isVisible={false} isOpen={this.props.isOpen} onClick={this.props.onRequestClose} />
+            </div>
+        );
+    }
+
+    renderChildrenWithFocusSet() {
+        const children = React.Children.toArray(this.props.children);
```

Comment:
We need to now if the centered child is disabled before the map processes the first child, to probably focus the first child (if its not disabled but the centered child is)